readthedocs/oauth/services/bitbucket.py

```diff
@@ -105,8 +105,10 @@ def create_repository(self, fields, privacy=None, organization=None):
         :rtype: RemoteRepository
         """
         privacy = privacy or settings.DEFAULT_PRIVACY_LEVEL
-        if ((privacy == 'private') or
-            (fields['is_private'] is False and privacy == 'public')):
+        if any([
```

Comment:
An `or` is more clear for me in these cases

Reply:
He, I decided to change it because it was hard to read to me because of the indentation :see_no_evil: